### ISSUE TITLE
Main menu shrinks after updating from official repos - Unacceptable behavior

### ISSUE THREAD

**Issue by GHNewbiee** [linuxmint/Cinnamon#7960]
```
 * Cinnamon version 3.6.7
 * Distribution - 18.3
 * Graphics hardware *and* driver used
  - Card-1: Advanced Micro Devices [AMD/ATI] BeaverCreek [Radeon HD 6520G]
  - Card-2: Advanced Micro Devices [AMD/ATI] Whistler [Radeon HD 6630M/6650M/6750M/7670M/7690M]
  - ati,radeon
 * 64 bit
 ```

**Issue**
The main menu shrinked/resized after updating from official repos through Mint Update Manager.

See the following photos:
- http://susepaste.org/70829743
- http://susepaste.org/82600834 after restoring defaults and reboot

Please, let me know if you have released any packages the last two days and especially yesterday.

**Steps to reproduce**
N/A

**Expected behaviour**
At least, restoring should be successful. On the contrary the things got worst!!!

**Other information**
**Unacceptable behavior** from a member of dev team which **hurts** the reliability of the whole team.

First of all the response: **`We aren't interested, this is a dev channel`** Well done! WELL DONE!

The team at least should have the kindness to answer my last question and inform about related packages which may have been released. No response after 5 minutes.

See photo http://susepaste.org/33099077 of my conversation with him/her.

Tia

**Comment by NikoKrause** [comment]
> Please, let me know if you have released any packages the last two days and especially yesterday.

Mint Update Manager -> View -> History of updates

**Comment by clefebvre** [comment]
I can't reproduce this observation.

https://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/

**Comment by NikoKrause** [comment]
>  Is there any "safe" way to restore the menu

The best way to restore a broken system is to restore your system with a timeshift snapshot to a previous working condition. This solution requires, that you have timeshift set up before the issue appeared.

Alternatively you can look at the following issue reports:
 https://github.com/linuxmint/Cinnamon/issues/5484 or https://github.com/linuxmint/Cinnamon/issues/3440

**Comment by GHNewbiee** [comment]
I usually use timeshift for big "jumps" eg system upgrade (I saved the system twice), but for small "steps" never. If I need a safety belt for walking around my home, better I stay sitting on sofa watching tv and eating popcorn for the rest of my life!

I have also checked the reports: I deleted some files but no positive results. It seems that "menu" had issues in the past and maybe in the present...

I think it is time to move to another home where I shall try to walk without safety belt!

Anyway, thanks for your time. I appreciate that!
